bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -54,6 +59,44 @@ async def watched_command(
         """
         await self.list_watched_users(ctx, oldest_first=oldest_first, update_cache=update_cache)
 
+    async def list_watched_users(
+        self, ctx: Context, oldest_first: bool = False, update_cache: bool = True
+    ) -> None:
+        """
+        Gives an overview of the nominated users list.
```

Comment:
```suggestion
        Gives an overview of the nominated users’ list.
```

Reply:
I think what I wrote here is correct, it's a list of nominated users, hence a nominated users list :D